ihatemoney/models.py

```diff
@@ -215,15 +216,25 @@ def exactmatch(self, credit, debts):
 
     def has_bills(self):
         """return if the project do have bills or not"""
-        return self.get_bills().count() > 0
+        return self.get_bills_unordered().count() > 0
 
-    def get_bills(self):
-        """Return the list of bills related to this project"""
+    def has_multiple_currencies(self):
+        """Return if multiple currencies are used"""
+        return self.get_bills_unordered().group_by(Bill.original_currency).count() > 1
```

Comment:
What happens if there are some bills with a currency like EUR, and some bills with no currency? I think it would still count as 2 with this function. But I have no idea what we should do in that case anyway...

Reply:
It is currently impossible to have this. When switching EUR -> XXX (no currency), all bills are converted to XXX.
In XXX default, it's only possible to add bills with XXX.
In any other default (e.g. EUR), it's only possible to add bills with a currency (e.g. **not** XXX).

I don't know if our test cases cover this from the API though.